### PR TITLE
Fix `cml ci` when called several times

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
`git config --unset <option>` exits with code 5 if the option doesn't exist (e.g. was already unset on a previous run)

> [_you try to unset an option which does not exist (ret=5)_](https://git-scm.com/docs/git-config)

### CODE REVIEW

**Comment by 0x2b3bfa0** [comment]
Bitbucket tests are red for unknown reasons. Retrying...

**Comment by DavidGOrtega** [comment]
> Bitbucket tests are red for unknown reasons. Retrying...

Looks like an unauthorised error. Might have changed the credentials?

**Comment by 0x2b3bfa0** [comment]
I haven't, at least. 🤔

File: src/drivers/bitbucket_cloud.js [hunk 1]
```diff
@@ -254,14 +254,14 @@ class BitbucketCloud {
...
+    git config --unset user.email;
+    git config --unset push.default;
     git config --unset http.http://${this.repo
       .replace('https://', '')
-      .replace('.git', '')}.proxy && \\
-    git config user.name "${userName || this.userName}" && \\
-    git config user.email "${userEmail || this.userEmail}" && \\
+      .replace('.git', '')}.proxy;
+    git config user.name "${userName || this.userName}" &&
+    git config user.email "${userEmail || this.userEmail}" &&
```
Comment (casperdcl) [inline src/drivers/bitbucket_cloud.js:264]: this was an accidental change, right?

```suggestion
    git config user.name "${userName || this.userName}" && \\
    git config user.email "${userEmail || this.userEmail}" && \\
```

File: src/drivers/bitbucket_cloud.test.js [hunk 2]
```diff
@@ -51,12 +51,12 @@ describe('Non Enviromental tests', () => {
...
-          git config --unset push.default && \\\\
-          git config --unset http.http://bitbucket.org/test/test.proxy && \\\\
-          git config user.name \\"john\\" && \\\\
-          git config user.email \\"john@test.com\\" && \\\\
+          git config --unset user.name;
+          git config --unset user.email;
+          git config --unset push.default;
+          git config --unset http.http://bitbucket.org/test/test.proxy;
+          git config user.name \\"john\\" &&
+          git config user.email \\"john@test.com\\" &&
```
Comment (casperdcl) [inline src/drivers/bitbucket_cloud.test.js:59]:
```suggestion
          git config user.name \\"john\\" && \\\\
          git config user.email \\"john@test.com\\" && \\\\
```

File: src/drivers/github.js [hunk 3]
```diff
@@ -565,9 +565,9 @@ class Github {
     repo.username = this.userName || userName;
 
     const command = `
-    git config --unset http.https://github.com/.extraheader && \\
-    git config user.name "${userName || this.userName}" && \\
-    git config user.email "${userEmail || this.userEmail}" && \\
+    git config --unset http.https://github.com/.extraheader;
+    git config user.name "${userName || this.userName}" &&
+    git config user.email "${userEmail || this.userEmail}" &&
```
Comment (casperdcl) [inline src/drivers/github.js:570]:
```suggestion
    git config user.name "${userName || this.userName}" && \\
    git config user.email "${userEmail || this.userEmail}" && \\
```

File: src/drivers/github.test.js [hunk 4]
```diff
@@ -46,9 +46,9 @@ describe('Non Enviromental tests', () => {
     const command = await client.updateGitConfig();
     expect(command).toMatchInlineSnapshot(`
       "
-          git config --unset http.https://github.com/.extraheader && \\\\
-          git config user.name \\"GitHub Action\\" && \\\\
-          git config user.email \\"action@github.com\\" && \\\\
+          git config --unset http.https://github.com/.extraheader;
+          git config user.name \\"GitHub Action\\" &&
+          git config user.email \\"action@github.com\\" &&
```
Comment (casperdcl) [inline src/drivers/github.test.js:51]:
```suggestion
          git config user.name \\"GitHub Action\\" && \\\\
          git config user.email \\"action@github.com\\" && \\\\
```

File: src/drivers/gitlab.js [hunk 5]
```diff
@@ -403,8 +403,8 @@ class Gitlab {
     repo.username = this.userName || userName;
 
     const command = `
-    git config user.name "${userName || this.userName}" && \\
-    git config user.email "${userEmail || this.userEmail}" && \\
+    git config user.name "${userName || this.userName}" &&
+    git config user.email "${userEmail || this.userEmail}" &&
```
Comment (casperdcl) [inline src/drivers/gitlab.js:407]:
```suggestion
    git config user.name "${userName || this.userName}" && \\
    git config user.email "${userEmail || this.userEmail}" && \\
```

File: src/drivers/gitlab.test.js [hunk 6]
```diff
@@ -54,8 +54,8 @@ describe('Non Enviromental tests', () => {
     });
     expect(command).toMatchInlineSnapshot(`
       "
-          git config user.name \\"john\\" && \\\\
-          git config user.email \\"john@test.com\\" && \\\\
+          git config user.name \\"john\\" &&
+          git config user.email \\"john@test.com\\" &&
```
Comment (casperdcl) [inline src/drivers/gitlab.test.js:58]:
```suggestion
          git config user.name \\"john\\" && \\\\
          git config user.email \\"john@test.com\\" && \\\\
```

**Review by casperdcl — COMMENTED** [review]
not sure about the removal of `\` - are they really unneeded?

**Comment by 0x2b3bfa0** [comment]
Link–only answer: https://unix.stackexchange.com/a/253520/107625 😄

**Comment by 0x2b3bfa0** [comment]
Sorry for not mentioning it in the pull request body: those changes are intentional, and backslashes are redundant.

**Review by casperdcl — APPROVED** [review]
ah ofc. ` && \n` not `\n &&`. icky but whatever :)

**Pull request merged**